src/edgepi/peripherals/spi.py

```diff
@@ -31,20 +30,33 @@ def __init__(
     ):
         self.devpath = f"/dev/spidev{bus_num}.{dev_id}"
         _logger.debug(f"Initialized SPI device with path '{self.devpath}'")
```

Comment:
move spi init log to spi_open

Reply:
[b34b401](https://github.com/EdgePi-Cloud/edgepi-python-sdk/pull/380/commits/b34b401442f857301b121e460db547b050157600)